### PR TITLE
Merge oort-core and oort-web into clouddriver-core and clouddriver-we…

### PR DESCRIPTION
**Pull request by duftler**
…b, respectively.
@ajordens or @cfieber please review.

### CODE REVIEW

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.groovy [hunk 1]
```diff
@@ -48,17 +55,24 @@ import javax.servlet.Filter
 @Configuration
 @Import([
   SimpleCORSFilter,
+  ApplicationsController,
```
Comment (ajordens) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.groovy]: I suspect scanning `com.netflix.spinnaker.clouddriver.controllers` is going to be a less cumbersome path forward than explicit @Imports.

File: settings.gradle [hunk 2]
```diff
@@ -18,7 +18,7 @@ rootProject.name="clouddriver"
 
 def subs = [
   kato: ["kato-web", "kato-aws", "kato-manual", "kato-jedis", "kato-docker", "kato-cf", "kato-titan", "kato-azure"],
-  oort: ["oort-core", "oort-web", "oort-aws", "oort-gce", "oort-bench", "oort-titan", "oort-cf", "oort-azure"],
+  oort: ["oort-web", "oort-aws", "oort-gce", "oort-bench", "oort-titan", "oort-cf", "oort-azure"],
```
Comment (ajordens) [inline settings.gradle]: does oort-web still exist, looks like it was folded into clouddriver-web?

Comment (duftler) [inline settings.gradle]: I've kept kato-web, mort-web and oort-web so the subprojects they bring in are not added directly into clouddriver-web. Especially for kato-web, it's a handful of dependencies. Thought it was cleaner this way, and can kill off the remaining {kato|mort|oort}-web .gradle files once the subprojects have all been themselves merged into clouddriver.

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.groovy (hunk 1 above)
Comment (duftler) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.groovy]: Done.

File: settings.gradle (hunk 2 above)
Comment (ajordens) [inline settings.gradle]: That's reasonable.

**Comment by ajordens** [comment]
LGTM

**Pull request merged**